src/main/resources/lib/reference-search/references-finder.ts

```diff
@@ -332,6 +333,43 @@ export class ReferencesFinder {
         return result;
     }
 
+    private findAlertInContextRefs = (content: ContentNode): QueryResult => {
+        if (content.type !== 'no.nav.navno:alert-in-context') {
+            return [];
+        }
+
+        const targetIds = forceArray(
+            content.data.target[content.data.target._selected].targetContent
+        );
```

Comment:
Kan vi legge på en sjekk at `content.data.target[content.data.target._selected]` er definert, og inneholder mer enn 0 elementer? I teorien skal den alltid være det ut fra schemaet for denne content typen, men just in case. 😅

Reply:
Enig!